src/shared/clients/ecsClient.ts

```diff
@@ -5,70 +5,69 @@
 
 import { ECS } from 'aws-sdk'
 import { ext } from '../extensionGlobals'
+import { getLogger } from '../logger'
 import { ClassToInterfaceType } from '../utilities/tsUtils'
 
 export type EcsClient = ClassToInterfaceType<DefaultEcsClient>
+
+export type ServicesAndToken =  {
```

Comment:
These types have the same general structure. A generic interface may work better here.